tests/Behat/DrupalContext.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_webtools\Behat;
+
+use Drupal\DrupalExtension\Context\RawDrupalContext;
+use Behat\Gherkin\Node\TableNode;
+
+/**
+ * Behat with module, feature and feature_set management.
+ */
+class DrupalContext extends RawDrupalContext {
+  /**
+   * A ModuleHandler instance.
+   *
+   * @var Drupal\Core\Extension\ModuleHandler
+   */
+  private $moduleHandler;
+
+  /**
+   * DrupalContext constructor.
+   */
+  public function __construct() {
+    $this->moduleHandler = \Drupal::moduleHandler();
+  }
+
+  /**
+   * Refresh the list of module.
+   *
+   * Before all scenario, we need to run module_list to refresh system_list,
+   * which is needed because of memory issues.
+   *
+   * @BeforeScenario
+   */
+  public function refreshDefaultEnabledModules() {
+    $this->moduleHandler->getModuleList();
+  }
+
+  /**
+   * Enables one or more modules.
+   *
+   * Provide modules data in the following format:
+   *
+   * | modules  |
+   * | blog     |
+   * | book     |
+   *
+   * @param \Behat\Gherkin\Node\TableNode $modules_table
+   *   The table listing modules.
+   *
+   * @Given the/these module/modules is/are enabled
```

Comment:
Step definition is wrong, as this doesn't enable the modules. Better:
Then I expect these modules to be enabled.

Reply:
Here it is checking if the module is enabled, on setUp it is where the module has been enabled.